Xamarin.Forms.ControlGallery.iOS/AppDelegate.cs

```diff
@@ -162,6 +162,8 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
 			MessagingCenter.Subscribe<NativeBindingGalleryPage>(this, NativeBindingGalleryPage.ReadyForNativeBindingsMessage, AddNativeBindings);
 
 			LoadApplication(app);
+			// uncomment to set default button title color for normal state
+			//UIButton.Appearance.SetTitleColor(UIColor.Red, UIControlState.Normal);
```

Comment:
@hartez I had added this line to update the proxy appearance, but I commented it out so that the button title color is not red across the gallery app. What would be another way to do this so that the test is scoped to `40251`? Perhaps use a custom renderer?